pkg/secretlock/local/standard_secret_lock.go

```diff
@@ -0,0 +1,140 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package local
+
+import (
+	"bytes"
+	"crypto/cipher"
+	"encoding/base64"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/btcsuite/btcutil/base58"
+	"github.com/google/tink/go/subtle/random"
+
+	"github.com/hyperledger/aries-framework-go/pkg/common/log"
+	"github.com/hyperledger/aries-framework-go/pkg/secretlock"
+)
+
+// package local provides a local secret lock service. The user must create a master key and store it
+// in a local file or an environment variable prior to using this service.
+//
+// A utility function is offered to help create an encrypted master key using a passphrase called:
+// 		CreateMasterKey(passphrase)
+//
+// The returned encrypted master key must be stored either in a file or an environment variable.
+// The user can then call either:
+//		InitFromPath(passphrase, path) or
+//		InitFromEnv(passphrase, envPrefix, keyURI)
+// to get an authenticated io.Reader instance needed to create a Lock service.
+//
+// To get the lock service, call:
+// 		NewServiceFromReader(passphrase, reader)
+// where reader is the instance returned from one of the previous Init functions.
+
+var logger = log.New("aries-framework/lock")
+
+// Lock is a secret lock service responsible for encrypting keys using a master key stored as environment variable.
+type Lock struct {
+	aead cipher.AEAD
+}
+
+// InitFromPath creates a new instance of a local secret lock Reader for an encrypted master key in path.
+func InitFromPath(passphrase, path string) (io.Reader, error) {
```

Comment:
These two "Init" functions could be renamed to something that alludes to their purpose - get the master key.

I suggest something like "MasterKeyFromPath"/"MasterKeyFromEnv" or something similar.

Reply:
Init for initializing the SecretLock, which happens to be fetching a reader of the master key to be used. If `MasterKey` makes more sense than `Init`, I'll rename the functions.

Reply:
`MasterKey` makes more sense to me.

Reply:
the functions are now renamed to `MasterKeyFromPath` and `MasterKeyFromEnv`